### ISSUE TITLE
Bogus line numbers for MEMORY_LEAK warnings

### ISSUE THREAD

**Issue by jsgf** [facebook/infer#83]
I'm getting reports of the form:
```
libecabal/options.c:1307: error: MEMORY_LEAK
   memory dynamically allocated by call to calloc() at line 1307, column 19 is not reachable after line 1307, column 19
```

But the file in question (options.c) only has 218 lines. Also the line number "1307" (and similar, 1309, etc) appear in MEMORY_LEAK warnings in other files (also with < 1300 lines). In fact, I can't see where calloc() is being called at all, let alone on a line 1307.

Compiling C code, using prebuilt infer-linux64-v0.1.0.tar.xz on a Debian wheezy system.

**Comment by jvillard** [comment]
Hi @jsgf, we believe this is fixed in the latest version [v0.1.1](https://github.com/facebook/infer/releases/tag/v0.1.1). Please re-open this issue if upgrading to v0.1.1 doesn't fix it for you.

**Comment by jsgf** [comment]
Yep, I noticed the update just after I pressed submit. Looks fixed.
